api/src/main/java/org/openmrs/module/registrationcore/api/mpi/pixpdq/PixPdqMessageUtil.java

```diff
@@ -224,20 +214,18 @@ else if(!pit.getUuid().equals(id.getAssigningAuthority().getUniversalID().getVal
                     pa.setCountyDistrict(xad.getCountyParishCode().getValue());
                     pa.setPostalCode(xad.getZipOrPostalCode().getValue());
                     pa.setStateProvince(xad.getStateOrProvince().getValue());
-                    if("H".equals(xad.getAddressType().getValue()))
+                    if ("H".equals(xad.getAddressType().getValue())) {
                         pa.setPreferred(true);
+                    }
 
                     patient.addAddress(pa);
-
                 }
 
                 // Mother's name
                 XPN momsName = pid.getMotherSMaidenName(0);
-                if(momsName != null)
-                {
+                if (momsName != null) {
                     PersonAttributeType momNameAtt = Context.getPersonService().getPersonAttributeTypeByName("Mother's Name");
-                    if(momNameAtt != null)
-                    {
+                    if (momNameAtt != null) {
```

Comment:
I appreciate the clean up, but avoid putting unrelated chages into PRs, it makes the whole thing longer and harder to review.